rust_sodium-sys/Cargo.toml

```diff
@@ -13,7 +13,7 @@ repository = "https://github.com/maidsafe/rust_sodium"
 version = "0.10.4"
 
 [dependencies]
-lazy_static = "~1.2.0"
+lazy_static = "~1.4.0"
```

Comment:
Could you change this to just `"1.4.0"` (i.e., remove the `~`)? That would make us compatible with all future 1.x versions which is probably what we want (assuming lazy_static follows semver).

Reply:
@madadam done

Reply:
Thanks!